pkg/azclient/utils/transport.go

```diff
@@ -47,5 +50,11 @@ func init() {
 				Renegotiation: tls.RenegotiateNever, // the same as default transport https://pkg.go.dev/crypto/tls#RenegotiationSupport
 			},
 		}
+
+		// Configure HTTP/2
+		if http2Transport, err := http2.ConfigureTransports(DefaultTransport); err == nil {
```

Comment:
https://cs.opensource.google/go/x/net/+/refs/tags/v0.35.0:http2/transport.go;l=275
 It seems that http2Transport doesn't take effect

Reply:
per my test, timeout would be 45s (without it, it would be 1m), so it does take effect